Extension/src/LanguageServer/client.ts

```diff
@@ -1294,21 +1298,19 @@ export class DefaultClient implements Client {
         this.rootFolder = workspaceFolder;
         this.rootRealPath = this.RootPath ? (fs.existsSync(this.RootPath) ? fs.realpathSync(this.RootPath) : this.RootPath) : "";
 
-        let storagePath: string | undefined;
-        if (util.extensionContext) {
-            const path: string | undefined = util.extensionContext.storageUri?.fsPath;
-            if (path) {
-                storagePath = path;
-            }
-        }
+        this.workspaceStoragePath = "";
```

Comment:
Since `this.workspaceStoragePath` is already getting assigned on line 1303 with either a value or empty value, then this first assignment can be removed.